releases.md

```diff
@@ -38,6 +38,65 @@ Further documentation available:
 - [Installing Tekton][tekton-installation]
 - Standard for [release notes][release-notes-standards]
 
+## Backwards Incompatible Changes
+
+In general we aim for new features to be added in a backwards compatible way,
+but sometimes we will need to make breaking changes. This policy outlines how we
+will make and communicate these.
+
+**NOTE**: Tekton Chains is working towards a formal beta release. Until then,
+all features are technically considered `alpha` (though we continue to do our
+best to retain backwards compatibility).
+
+### Stability levels
+
+| Stability Level | Deprecation Window                                       |
+| --------------- | -------------------------------------------------------- |
+| alpha           | none                                                     |
+| beta            | 3 months or 3 minor releases (which ever is longer)      |
```

Comment:
How often are we creating minor releases?

Doesn't this sound a little aggressive for `beta`? At least increase the deprecation window to 6 months.

Reply:
> How often are we creating minor releases?

We aim for ~monthly releases.

> Doesn't this sound a little aggressive for beta? At least increase the deprecation window to 6 months.

A quarter heads up feels reasonable to me as a heads up that some thing not in stable is changing. This is also a minimum window (i.e. chains has technically not had a deprecation policy and been under alpha, but we've rarely made breaking changes). We can re-evaluate this in the future if needed - extending a deprecation window is a backwards compatible change.